README.md

```diff
@@ -1,58 +1,78 @@
 
-Found a problem?  Use the issues tab above to raise an issue.  Please include
-all the information requested in the issue template, unless you're certain it's
-not relevant.
+### Links
+- **[GriefPrevention Spigot.org](https://www.spigotmc.org/resources/griefprevention.1884/)**
+- **[GriefPrevention Bukkit.org](https://dev.bukkit.org/projects/grief-prevention)**
+- **[![Latest Release](https://github.com/TechFortress/GriefPrevention/workflows/Releases/badge.svg)](https://github.com/TechFortress/GriefPrevention/releases/latest)**
+- **[![Latest Build](https://github.com/TechFortress/GriefPrevention/workflows/Build/badge.svg)](https://github.com/TechFortress/GriefPrevention/actions?query=workflow%3ABuild)**
+- **[Wiki](https://github.com/TechFortress/GriefPrevention/wiki)**
 
 ### Wiki
-
 The [GriefPrevention wiki here on GitHub](https://github.com/TechFortress/GriefPrevention/wiki)
 contains documentation on using Grief Prevention, its range of commands and
 permissions, etc.
 
-### See also
+## IMPORTANT
+
+### Requirements:
+- Java 1.8 or higher
+- CraftBukkit (1.15.*)
 
-[Grief Prevention on spigotmc.org](https://www.spigotmc.org/resources/griefprevention.1884/)
+### Configurate
+2. Download [Grief Prevention](https://github.com/TechFortress/GriefPrevention/releases/latest)
+3. Restart your minecraft server
+4. That was it.
 
-[Grief Prevention on dev.bukkit.org](https://dev.bukkit.org/projects/grief-prevention)
+## Developer
 
+### Clone
+1. Clone this repo "git clone https://github.com/TechFortress/GriefPrevention.git"
+2. Open eclipse and right click on the "Project Explorer"
+3. Click "Import..."
+4. Maven -> Existing Maven Projects
+5. Select the downloaded repo
+6. Finished
 
-### Adding GriefPrevention as a gradle/maven/sbt/leiningen dependency
+## Build
+1. Click right click on the **GriefPrevention** folder and select "Run as" -> "Maven Build..."
+2. Put into Goals this "clean install compile package -Dgriefprevention-version=17.0.0"
+3. Click Run
+4. Your jar will be builded under the folder "target"
+5. Finished
 
-Apparently there's this neat thing called JitPack [![](https://jitpack.io/v/TechFortress/GriefPrevention.svg)](https://jitpack.io/#TechFortress/GriefPrevention) that makes a public maven repo for public Github repos on the fly.
-According to it, this is all you need to do to add this to your maven project:
+## Release a new version
+1. git tag **version**
+2. git push origin **version**
+3. Finished
+
+### Maven
 ```xml
-	<repositories>
+<repositories>
 		<repository>
 		    <id>jitpack.io</id>
 		    <url>https://jitpack.io</url>
 		</repository>
-	</repositories>
-```
-
-Replace `<version>` number with this number: [![](https://jitpack.io/v/TechFortress/GriefPrevention.svg)](https://jitpack.io/#TechFortress/GriefPrevention)
```

Comment:
Not sure why you'd remove this - it'd be nice to not have to constantly update the readme because of version bumps.

Reply:
Okay